src/main/java/uk/gov/justice/hmpps/prison/api/model/Agency.java

```diff
@@ -31,4 +34,9 @@
 
     @ApiModelProperty(value = "Agency is active", example = "true")
     private boolean active;
+
+    @ApiModelProperty(value = "Establishment types for the agency", example = "IM")
```

Comment:
Maybe add additional description of an agency type, it still means nothing to me. Does in indicate if this is a women's prison for instance?

Reply:
Agreed and really  that was originally what I intended.  So I had some JPA mapping issues which I never found a clean solution to.  This supporting attribute for this is actually a reference code.  In this case it forms part of the composite key on the new JPA entity AgencyLocationEstablishment.  The particular entity attribute would be using a JoinColumnOrFormula annotation and JPA does not this when part of a composite key, hence I couldn't pull in the description.  So we can (at least) get the description from the reference code endpoint (for now).